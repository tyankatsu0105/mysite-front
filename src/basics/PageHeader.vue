<template>
	<h1 class="PageHeader">
		{{ text }}
	</h1>
</template>

<script>
export default {
  props: {
    text: {
      type: String,
      requirer: true
    }
  }
};
</script>

<style scoped lang='scss'>
.PageHeader {
  margin-bottom: 40px;
  font-size: 3rem;
  font-weight: bold;
  text-align: center;
  text-transform: uppercase;
  letter-spacing: 0.3em;
}
</style>
