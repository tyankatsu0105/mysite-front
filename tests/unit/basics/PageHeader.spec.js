import { shallowMount } from "@vue/test-utils";
import PageHeader from "@/basics/PageHeader.vue";

const propsData = {
  text: "aaaaaaaa"
};

describe("PageHeader", () => {
  it("render", () => {
    const wrapper = shallowMount(PageHeader, {
      propsData
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
