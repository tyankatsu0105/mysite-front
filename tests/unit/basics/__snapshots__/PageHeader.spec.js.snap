// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader render 1`] = `
<h1 class="PageHeader">
  aaaaaaaa
</h1>
`;
